xbmc/video/dialogs/GUIDialogAudioSubtitleSettings.cpp

```diff
@@ -156,6 +158,16 @@ void CGUIDialogAudioSubtitleSettings::OnSettingChanged(const CSetting *setting)
       g_application.m_pPlayer->SetAudioStream(m_audioStream);    // Set the audio stream to the one selected
     }
   }
+  else if (settingId == SETTING_VIDEO_STREAM)
```

Comment:
Agree, but it feels wrong in the video setting dialogue. I would expect a dialogue with all stream settings.